### ISSUE TITLE
Comment

### ISSUE THREAD

**Issue by rajesh06** [PirateGrunt/data_complete#1]
Hi Brian - I realize that you want to support multiple languages, but perhaps it's not necessary to support multiple packages within each language. For example, under `Data frames`, you support `tidyverse` but not `data.table`. Just a thought.

**Comment by PirateGrunt** [comment]
Fair point and reflective of my own bias. `data.table` looks like a great package, I've simply never made the time to use it much.
